pandas/tests/frame/test_axis_select_reindex.py

```diff
@@ -36,352 +33,6 @@ def test_merge_join_different_levels(self):
             result = df1.join(df2, on="a")
```

Comment:
what is left in this file? also some aligns left. i would rather have you simply rename this an extract the non-reindex tests (which seems to be very few)

Reply:
> also some aligns left.

theres one with "align" in the name but its testing arithmetic, not the align method directly.